code/index.html

```diff
@@ -1,22 +1,109 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto&display=swap" rel="stylesheet">
+    <title>Sunrise Travel</title>
+</head>
+
+<body>
+
     <header>
-      <!-- Some kind of logo and navbar -->
+        <!-- logo and navbar -->
+        <img src="./pics/logo.png" alt="logo">
+
+        <div class="hamburger">
+            <div></div>
+            <div></div>
+            <div></div>
+        </div>
+
+        <nav>
+            <a href="#">Contact</a>
+            <a href="#">Destinations</a>
+            <a href="#">Booking</a>
+        </nav>
     </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
+
+
+    <section class="info">
+        <!-- <div class="image"> -->
+        <img src="pics/fiji.jpg" alt="vaccination image">
+        <!-- </div> -->
+        <div class="text">
+            <h1>Find deals on hotels, flights <br>and package trips.</h1>
+            <h3><i>We have destinations all over the world.</i></h3>
+            <button class="btn">Book here</button>
+        </div>
     </section>
 
-    <section class="">
-      <!-- The other news  -->
+    <section class="destinations">
+        <article class="destination-box">
+            <h2>Greece</h2>
+            <img src="pics/greece.jfif" alt="Greece">
+        </article>
+        <article class="destination-box">
+            <h2>Spain</h2>
+            <img src="pics/spain.jfif" alt="Spain">
+        </article>
+        <article class="destination-box">
+            <h2>Italy</h2>
+            <img src="pics/italy.jpg" alt="wheather image">
+
+        </article>
+        <article class="destination-box">
+            <h2>Carribean</h2>
+            <img src="./pics/carribean.jpg" alt="Carribean">
+        </article>
+        <article class="destination-box">
+            <h2>South America</h2>
+            <img src="./pics/southamerica.jfif" alt="South America">
+        </article>
+        <article class="destination-box">
+            <h2>Africa</h2>
+            <img src="./pics/africa.jfif" alt="Africa">
+        </article>
+        <article class="destination-box">
+            <h2>Asia</h2>
+            <img src="./pics/asia.jfif" alt="Asia">
+        </article>
+        <article class="destination-box">
+            <h2>Cities</h2>
+            <img src="./pics/newyork.jpg" alt="New York">
+        </article>
+    </section>
```

Comment:
This is well structured and nicely indented, great job ⭐️ Perhaps for next time just swap the `<h3>`-tag you used higher up with the `<h2`>-tag you have here. It's more semantically correct to go from highest to lowest.